### PR TITLE
Update tado.markdown

### PR DESCRIPTION
**Pull request by samnewman86**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @samnewman86, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16297).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by samnewman86** [comment]
The documentation docs dont seem to provide a standard for deprecation?

**Review by frenck — CHANGES_REQUESTED** [review]
There are unrelate commits in your PR. Please clean it up so only the contents (your commits) are in this PR as a change.

**Comment by samnewman86** [comment]
> There are unrelate commits in your PR. Please clean it up so only the contents (your commits) are in this PR as a change.

Sorry, no idea how to do that

**Comment by frenck** [comment]
You'll need to run a git rebase to put it on top of the next branch.

Could you also update the PR description to have a link to the upstream code PR that deprecates it in the code base?

Thanks 👍

File: source/_integrations/tado.markdown [hunk 1]
```diff
@@ -31,6 +31,8 @@ There is currently support for the following device types within Home Assistant:
 
 To use your Tado thermostats in your installation, go to **Configuration** >> **Integrations** in the UI, click the button with `+` sign and from the list of integrations select **Tado**.
 
+Previously it was necessary to add your tado username and password to your `configuration.yaml` file, this is no longer necessary and should be removed.
```
Comment (klaasnicolaas) [inline source/_integrations/tado.markdown]:
```suggestion
Previously it was necessary to add your Tado username and password to your `configuration.yaml` file, this is no longer necessary and should be removed.
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
This will fix the build error

**Review by samnewman86 — COMMENTED** [review]
Thanks

File: source/_integrations/tado.markdown [hunk 2]
```diff
@@ -31,6 +31,8 @@ There is currently support for the following device types within Home Assistant:
 
 To use your Tado thermostats in your installation, go to **Configuration** >> **Integrations** in the UI, click the button with `+` sign and from the list of integrations select **Tado**.
 
+Previously it was necessary to add your Tado username and password to your `configuration.yaml` file, this is no longer necessary and should be removed.
```
Comment (frenck) [inline source/_integrations/tado.markdown:34]: This should not be documented. We document the current state of things; this specific log belong in the release notes.

**Comment by frenck** [comment]
Closing this PR, as the added documentation should not be in our docs. Nevertheless, thanks for willing to contribute 👍

**Pull request closed without merging**